python/pen_checker_cdc.py

```diff
@@ -56,6 +56,7 @@ def get_options():
     parser.add_argument('--aln', required=True, help='Gene alignment  (required)', type = str)
     parser.add_argument('--tlength', required=True, help='Target length of TPD domain  (required)', type = int)
     parser.add_argument('--tolerance', help='Deviation from expected length allowed', type = int, default = 10)
+    parser.add_argument('--data_dir', help='Location of the data directory of package', type = str, default="./data")
```

Comment:
Adding in option for data_directory to allow for running outside the main repo directory